formats/hocon/src/test/kotlin/kotlinx/serialization/hocon/HoconMapTest.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.hocon
+
+import com.typesafe.config.ConfigFactory
+import kotlinx.serialization.Serializable
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+private const val CONFIG_FOR_ROOT_MAP = """
+key1 {
+    a = "text1"
+    b = 11
+    }
+
+key2 {
+    a = "text2"
+    b = 12
+}
+
+key3 {
+    a = "text3"
+    b = 13
+}
+"""
+
+
+class HoconRootMapTest {
+    @Serializable
+    data class CompositeValue(
+            val a: String,
+            val b: Int
+    )
+
+    @Test
+    fun `config with root map`() {
```

Comment:
Backticked tests are legacy from the initial versions. We stick to `testFoo` naming convention.